rpm/del-fixtures-sign.sh

```diff
@@ -0,0 +1,44 @@
+#!/usr/bin/env sh
+#
+# Delete signatures of all packages recursively in target directory.
+# One RPM utility `rpmsign` must be installed.
+
+set -euo pipefail
+#------------------------------------------------------------------------------
+# Helper Functions
+#------------------------------------------------------------------------------
+
+# Print a message to stdout explaining how to use this script.
+#
+# NOTE: $0 corresponds to the the script's name only in some shells and in some
+# contexts.
+show_help() {
+fmt <<EOF
+usage: del-fixtures-sign.sh <target_dir>
+
+Delete signatures of all packages recursively in <target_dir>
+
+EOF
+cat <<EOF
+<target_dir>
+    The directory from which package's signature will be removed.
+EOF
+}
+
+# Fetch target_dir from user.
+if [ "$#" -lt 1 ]; then
+    echo 1>&2 'Error: Too few arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+elif [ "$#" -gt 1 ]; then
+    echo 1>&2 'Error: Too many arguments received.'
+    echo 1>&2
+    show_help 1>&2
+    exit 1
+else
+    target_dir="$(realpath --canonicalize-missing "${1}")"
+fi
+
+# Delete signatures from packages in target_dir recursively
+rpmsign --delsign `find ${target_dir} -print | grep -i '.*[.][sd]*rpm'`
```

Comment:
Do not use backticks. They're deprecated in favor of `$()`, not least because they cannot be nested.